release/docker/package_docker

```diff
@@ -55,7 +55,10 @@ image_name="memgraph:${version}"
 image_package_name="memgraph-${version}-docker.tar.gz"
 
 # Build docker image.
-docker build -t ${image_name} ${tag_latest} -f ${dockerfile_path} --build-arg release=${package_name}.${extension} .
+docker build -t ${image_name} ${tag_latest} -f ${dockerfile_path} \
+--build-arg BINARY_NAME=${package_name} \
+--build-arg EXTENSION=${extension} \
+--build-arg TARGETARCH="".
```

Comment:
After consistently using the architecture related postfix in CMake, I am suspicious about the empty `TARGETARCH` parameter here. I think the name of package will be the default one coming from CMake. If I am wrong, could you please describe how that this works?

Reply:
Furthermore this script is called from two places:
1. release_docker.yaml which download the binary packages from our website.
2. from `release/package/run.sh` which uses a freshly built binary package that is named by CMake.

Wouldn't that cause an issue?

Reply:
All that you said is valid but in this script during extraction of `package_name` and `extension`, the architecture is a part of binary name => `package_name`, therefore we pass an empty string to ignore it, but in the docker case the architecture is passed by the docker build extension `build` and that how we decide internally inside Dockerfile for which architecture we are building. E.g. running this script like this:
` ./package_docker --latest /home/bajic/Downloads/memgraph_2.2.1-1_amd64.deb`
`package_name` = `memgraph_2.2.1-1_amd64`
`extension` = `deb`

Reply:
The test run can be seen here https://github.com/memgraph/memgraph/actions/runs/2076704439